src/cordial_gui/web/src/css/low_vision_gui_input_only.css

```diff
@@ -18,3 +18,19 @@
     margin: 1vh;
     padding: 2px;
 }
+
+#col-1-numpad {
+    padding: 0.5em;
```

Comment:
You may want to use `vh` or `vw` (view width and view height) to make the content scale to the screen. It may be fine, but what you make may look different on your preview and on the tablet used.